plugins/ircbridge/ircbridge.js

```diff
@@ -589,13 +621,45 @@ function setAlias(telegramGroup, telegramUser, alias) {
         });
         return;
     }
-    storage.telegramAliases[telegramUser] = alias;
+    setUserPreference(telegramUser, PREFERENCES_KEY_ALIAS, alias);
     api.sendMessage({
         chat_id: telegramGroup,
         text: 'Alias set to "' + alias + '".'
     });
 }
 
+function setUserRelayable(telegramGroup, telegramUser, relayable) {
+    const channels = findChannelsForGroup(telegramGroup);
+    
+    if (!relayable) {
+        const messagesBeingRelayed = getUserPreference(telegramUser, PREFERENCES_KEY_RELAYABLE);
+        const message = messagesBeingRelayed ?
+            `Your messages are currently being relayed to ${channels.join(', ')}.` :
+            'Your messages are currently NOT being relayed.';
+        return api.sendMessage({
+            chat_id: telegramGroup,
+            text: message
+        });
+    } else if (YES.includes(relayable.toLowerCase())) {
```

Comment:
Why not just use `Boolean(relatable to lower case())`?

Reply:
```
> Boolean('false')
true
> Boolean('true')
true
> Boolean('this should be rejected')
true
> Boolean('0')
true
> Boolean(0)
false
```

Reply:
I forgot JavaScript is shitty :')

Reply:
Tbh having `Boolean('false')` evaluate to `false` would be retarded

Reply:
This behavior is what I would expect from a sane language (of course PHP has `'0' == false IIRC)

Reply:
Well if you're coersing 'false' i'dn't expect it to yield `true`. I agree it shouldn't be falsy though. that'd indeed be retarted.